Modelica/Thermal/HeatTransfer/Rankine/TemperatureSensor.mo

```diff
@@ -3,41 +3,10 @@ model TemperatureSensor "Absolute temperature sensor in degRankine"
 
   Modelica.Blocks.Interfaces.RealOutput T annotation (Placement(
         transformation(extent={{90,-10},{110,10}})));
-  Interfaces.HeatPort_a port annotation (Placement(transformation(extent={{
-            -110,-10},{-90,10}})));
```

Comment:
Removing the temperature sensor?

Reply:
Re-introduce heat port in Rankine.TemperatureSensor in 92975e5e94e47497f4e0fd4a5515b5ed92c043b1